### PR TITLE
Use the stable rust toolchain instead of a specific version

### PR DESCRIPTION
**Pull request by jedisct1**

### CODE REVIEW

**Comment by avidal** [comment]
Yeah, I think this is fine. I only have it at a specific version because that's what the fastly template does; but I think they have plans to remove the specific toolchain bit soon anyway since it works on stable.

File: testdata/rust-toolchain [hunk 1]
```diff
@@ -1 +1 @@
-1.47.0
+stable
```
Comment (avidal) [inline testdata/rust-toolchain:1]: We can just delete this file now

**Review by avidal — CHANGES_REQUESTED** [review]

**Comment by jedisct1** [comment]
If it wasn't the case before, and now that Rust 1.48.0 has been released, it is likely that the Fastly crate works on stable.

File: testdata/rust-toolchain (hunk 1 above)
Comment (jedisct1) [reply testdata/rust-toolchain:1]: Sure.

Comment (jedisct1) [reply testdata/rust-toolchain]: Done!

**Review by avidal — APPROVED** [review]

**Pull request merged**